### DISCUSSION TITLE
Amazon order ID is changing

### DISCUSSION THREAD

**Discussion by jeremy-dragonfly** [amzn/selling-partner-api-models#3469] [General]
Hi,

I'm seeing that as of Jan 8th 2023, many order item IDs are changing. This is in the "XML Orders By Last Update Report", and we're seeing that for a given order ID, the single order item ID within it has changed.

We've never seen this happen before, and we're using the order item ID to link orders and shipments and financial reports. So we're not sure whether this is intentional or some kind of error, and if it's temporary or not. We'd be interested to hear whether anyone else is experiencing this.

Regards,
Jeremy

**Comment by ForrestWang** [comment]
We encountered this same issue too, it hurts our business A LOT. please fix it ASAP.

**Reply by tomaszhanc** [reply]
We also have that issue. There is a bug opened for it here: https://github.com/amzn/selling-partner-api-docs/issues/2848
